src/classiceditorui.js

```diff
@@ -148,7 +149,9 @@ export default class ClassicEditorUI extends EditorUI {
 			view.stickyPanel.viewportTopOffset = this._toolbarConfig.viewportTopOffset;
 		}
 
-		view.toolbar.fillFromConfig( this._toolbarConfig.items, this.componentFactory );
+		toolbar.fillFromConfig( this._toolbarConfig.items, this.componentFactory );
+
+		toolbar.label = editor.t( this._toolbarConfig.label );
```

Comment:
There must always be a string inside `t()`. And you must not use it as `editor.t()`. It should always be `t()`.